packages/react-components/src/components/RichTextEditor/Plugins/CopyPastePlugin.tsx

```diff
@@ -41,7 +33,7 @@ export default class CopyPastePlugin implements EditorPlugin {
  */
 export const removeImageElement = (event: PluginEvent): void => {
   // We don't support the pasting options such as paste as image yet.
-  if (event.eventType === CompatiblePluginEventType.BeforePaste && event.pasteType === CompatiblePasteType.Normal) {
+  if (event.eventType === 'beforePaste' && event.pasteType === 'normal') {
```

Comment:
I'm guessing the new content mode doesn't provide a type for paste events anymore? Should we have some internal types instead of leave them as strings here?

Reply:
Added! Please note that I haven't added an enum for `event.pasteType` as there is a type check for it and VSCode will show a warning if the value is incorrect. The values allowed by RoosterJS:
![image](https://github.com/Azure/communication-ui-library/assets/98852890/c73bc6e4-9c39-4b67-b40b-3359cb4c3308)